### DISCUSSION TITLE
trying to get used to hitting shift : so many times, why cant we use ; = no shift

### DISCUSSION THREAD

**Discussion by webbb82** [helix-editor/helix#6309] [General]
so im pretty new to vim helix way of typing and im getting use to it slowly. I just feel like a few changed would make the whole workflow smoother. So for 1

### Changing the ESC Key
1. Change the ESC key function to something lower on the keyboard like the function key or whatever. Having to travel all the way up  to escape 600 time  is a journey.

### The : key 
2. **Changing the** : **function to** ;, now hear me out why would we be made to press shift when we dont have to? is there a technical reason behind this? I tried to change it in my bashrc file but that didnt work lol.

**Yea just 2 things.** Those two and I feel thing would be smoother. But im new maybe you get used to making the treck a inch and a half up to the ESC key, I guess im just bitching but are these things you change or just deal with? and why would they chose those out of all the keys. why : over ; 
Im going to make a tshirt and start a campaign. "love the ; ditch the : "

**ALso when you navigate around do you use both hands to press the H,J,K,L keys or does your left hand control h j and the right is j k ? .**

**Comment by archseer** [comment]
> Change the ESC key function to something lower on the keyboard like the function key or whatever. Having to travel all the way up to escape 600 time is a journey.

Remap your caps lock to escape.

> Changing the : function to ;

`:` as command mode has been used in many programs for ages. I should be far less common to type too. You can remap this in your user config.

**Reply by webbb82** [reply]
ya thanks. I feel like i just need to get use to it. _

im curious_ to hit shift, do you use your left hand shift or right hand and like use your ring finger for right shift then like middle or index for the :? i feel like this is going to be similar to playing guitar chords. 

_`I did find a ton of new menus and features today when i was using it. Ive forced myself to only use that from now on.`_ 

_Oh one last question you dont have to answer these_ but to move around do you use the arrow keys or do you use hjkl?  and if you use hjkl do you use only your right hand? kinda feels like thats the way to go.

**Reply by diktomat** [reply]
Just use the fingers you would use when touch-typing:
- `:` = Left pinky shift, right pinky ;
- `hjkl` = right index, middle, and ring finger

That's the appeal of modal editing, your hands don't have to move between typing and hitting commands. Also, note that there are much more movement keys beyond hjkl, they aren't commonly used that often, despite being the first thing newcomers to modal editing learn.

**Reply by webbb82** [reply]
Ya know I never considered doing it that way left hand shift and right hand : . I do that for other combos.  I went through the tutorial thing they have and yanked all the good stuff for reference. Ya know what would be really nice. If there was an app that depending on the software you are using would show you a cheatsheet. So when you are in helix you hit some keybind and a popup window with helix keybinds shows up or if you are in vim  hit the same keybind and a vim keybind cheatsheet appears.... I think I know my mission now to program this. sorry I just had this idea as i was typing . anyway thanks again
